README.md

```diff
@@ -31,86 +25,186 @@ Templates in this repository include:
 - Web blog with **serverside rendering** and **import statements** (`blog`)
 - A "Contact Us" function with a **secret password** (`contact-us`)
 
-See [project structure](#project-structure) to learn about repository and see how to [deploy](#deploy-a-function) your function.
+See [templates structure](#templates-structure) to learn about repository and
+see how to [deploy](#deploy-a-function) your function.
 
-## Project structure
- Directory structure is as follows: **root/[language]/[template]** where *root* is 
- the github repository itself, *language* is the programming language used and
- *template* is the name of the template. Function is created using templates via
- `--repository` flag when using `func create` command. More in [How-To-Use](#how-to-use) section.
+## Prerequisites
+In order to use Functions, you will need a few things:
 
- ```
-github.com/gauron99/func-templates <--[root]
-├── go <------------------------------[language]
-│   └── hello <-----------------------[template]
-│       └── function source files
-│   └── splash-screen
-│       └── ...   
-├── node
-│   └── hello
-│   └── splash-screen
-├── ...
+### Download the `func` binary
+You can clone our [GitHub repository](https://github.com/knative/func/) and 
+build your binary from the source or download it straight from the
+[release pages](https://github.com/knative/func/releases) under *Assets*.
+
+Alternatively, for your convenience, see if your OS and architecture can be
+found in the list below for an easy copy&paste and download our latest version
+`knative-v1.16.1`.
+
+### Linux
+
+#### amd64
+```bash
+curl -L -o /usr/local/bin/func https://github.com/knative/func/releases/download/knative-v1.16.1/func_linux_amd64
```

Comment:
Is there a floating "latest" available so we don't have to update this on each release?